### DISCUSSION TITLE
Help us name the 1.5 release

### DISCUSSION THREAD

**Discussion by PedramNavid** [dagster-io/dagster#16662] [Community]
Hi everyone! Pedram from the Dagster team here.

We'll be releasing Dagster v1.5 soon and want your feedback!

Every major and minor release gets named after a hit single or an album from a pop diva. Sometimes, the name of the song/album is inspired by what changes are coming with the release.

For inspiration, 1.5 will have:

[Asset Checks](https://github.com/dagster-io/dagster/discussions/15880)
[Dagster External Execution Protocol ](https://github.com/dagster-io/dagster/discussions/16319)
Asset Spec and Asset Dep
Among other things...

And here are some songs/albums we've previously named our versions after!

[Material Girl](https://www.youtube.com/watch?v=xeddEMuM0dA) (Madonna)
[Smooth Operator](https://www.youtube.com/watch?v=4TYv2PhG89A) (Sade)
[Formation](https://www.youtube.com/watch?v=WDZJPJV__bQ) (Beyonce)
[Thank U, Next](https://www.youtube.com/watch?v=gl1aHhXnN1k) (Ariana Grande)
[Hello](https://www.youtube.com/watch?v=YQHsXMglC9A) (Adele)
[Cool for the Summer](https://www.youtube.com/watch?v=il9nqWw9W3Y) (Demi Lovato)
[Lucky Star](https://www.youtube.com/watch?v=ThHz9wlBeLU) (Madonna)
[The Edge of Glory](https://www.youtube.com/watch?v=QeWBS0JBNzQ) (Lady Gaga)
[Laundry Service](https://www.youtube.com/watch?v=7BPE5sR6oUU&list=PL035E8452800ACEA5) (Shakira)
[In the Zone](https://www.youtube.com/watch?v=we8h_2RFAyA) (Britney Spears)
[Waiting to Exhale](https://www.youtube.com/watch?v=_FT7ciAPuJk&list=PL_sN_rrP4od0gZ4C3LGkgcaRpVa_koQX4) (Whitney Houston, et al.)

[Listen to the full playlist](https://open.spotify.com/playlist/476UFttwzU8476ki1Aqcdd)

Drop a comment with your suggestions, and upvote the suggestions you like!

**Comment by abkfenris** [comment]
[Sweet Dreams (Are Made of This)](https://www.youtube.com/watch?v=qeMFqkcPYcg) (Annie Lennox)

**Comment by alangenfeld** [comment]
Straight Up (Paula Abdul)

**Comment by mjclarke94** [comment]
Given the asset checks release:
How Will I Know - Whitney Houston

(that or "It's not right but it's okay"!)

**Reply by tacastillo** [reply]
I'm upset at how much I like this one.

**Reply by PedramNavid** [reply]
"How will I know? Don't trust your feelings...I say a prayer with every heart beat" this is really good.

**Comment by askvinni** [comment]
Both because of asset checks and external execution protocol, this is gonna open up so many possibilities, so it's only natural to go with:

Gimme More - Britney Spears
